src/main/java/pages/checkoutConfirmationPage/ConfirmProductsTable.java

```diff
@@ -0,0 +1,50 @@
+package pages.checkoutConfirmationPage;
+
+import org.openqa.selenium.By;
+import pages.BasePage;
+
+import java.math.BigDecimal;
+
+import static engine.drivers.WebDriverFactory.getWebDriverInstance;
+import static enums.Currency.convertCurrencyFromCookie;
+
+public class ConfirmProductsTable {
```

Comment:
Why we need this table to be part of this PR?

Reply:
The table is one of the tables where the price is located and where I can use ``convertCurrencyFromCookie(totalPriceText);``.